GPU/Debugger/Debugger.cpp

```diff
@@ -111,7 +111,7 @@ bool NotifyCommand(u32 pc) {
 	}
 
 	bool process = true;
-	if (cmd == GE_CMD_PRIM || cmd == GE_CMD_BEZIER || cmd == GE_CMD_SPLINE) {
+	if (cmd == GE_CMD_PRIM || cmd == GE_CMD_BEZIER || cmd == GE_CMD_SPLINE || cmd == GE_CMD_TRANSFERSTART) {
```

Comment:
I think it might be fair to call it a prim if we highlight the area it's uploading to and it actually affects the target framebuffer.  Otherwise, I think this is confusing.  I think it might make sense to add a new step type, but a LOT of people use the GE debugger to rom hack and investigate graphics, and seeing a bunch of prim steps that highlight nothing is a confusing experience.

-[Unknown]

Reply:
I'm gonna revert this for now. I do want the functionality though so will do it in some optional way.

Reply:
Maybe it can be under Breakpoints, "Break on block transfers"?  I would kinda want some way to know when it's starting a list and a memcpy happened too, because I agree this sort of thing is useful when debugging.  I'm most interested in block transferred from/to the render target (color, depth), although textures are interesting too (but less so, depending.)

-[Unknown]

Reply:
Yeah, could be okay there. Can also think of some other additional break types that could be useful (break on through mode, for example, to skip to HUD/post processing after 3D in some games, for example).

Reply:
Well, you can already break on the vertex type changing.  It'd be nice to filter those by bits, that feels more like a condition on the existing register-based breakpoints.

-[Unknown]